.ci/targets/web_platform_tests.yaml

```diff
@@ -6,9 +6,15 @@ tasks:
     script: .ci/scripts/tool_runner.sh
     args: ["fetch-deps", "--web", "--supporting-target-platforms-only"]
     infra_step: true
-  - name: build examples
-    script: .ci/scripts/tool_runner.sh
-    args: ["build-examples", "--web"]
+  # Is this needed?
```

Comment:
Can you either fully remove this or file an issue and put a TODO in here?

Reply:
Yes, I'm going to fully remove it. I'm also looking at running the unit tests in wasm.